docs/versioned_docs/version-zenith/guides/snippets/github-google-cloud/go/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"context"
+	"strings"
+
+	run "cloud.google.com/go/run/apiv2"
+	runpb "cloud.google.com/go/run/apiv2/runpb"
+	"dagger.io/dagger/dag"
```

Comment:
Don't think we need this import - the `dag` var comes from `dagger.gen.go`

Reply:
Fixed, thanks!